politeiawww/api/v1/api.md

```diff
@@ -704,8 +706,10 @@ Request:
 
 ```json
 {
-  "token": "337fc4762dac6bbe11d3d0130f33a09978004b190e6ebbbde9312ac63f223527",
-  "status": 4
+  "proposalstatus": 3,
```

Comment:
The key should just be called `status`.

Reply:
Actually, `proposalstatus` is correct. But the table definition above is wrong which is now fixed.

Reply:
@AlanL1 can you point to the code that calls it `proposalstatus`?

Reply:
https://github.com/decred/politeia/blob/30cc2c072c714c9de2ba73392591175087aae3b2/politeiawww/www.go#L514-L536

handleSetProposalStatus() get `proposalstatus` from json (defined within the struct SetProposalStatus in v1.go which I modified), then call ProcessSetProposalStatus()  to do the actual modification.